src/project/cache/flowr-analyzer-cache.ts

```diff
@@ -185,10 +185,19 @@ export class FlowrAnalyzerCache<Parser extends KnownParser> extends FlowrCache<A
 		if(useDataflow) {
 			/* if force is active, it will have triggered with normalize */
 			dataflow = await this.dataflow();
+		} else {
+			dataflow = this.peekDataflow();
```

Comment:
peeks should not be automatic but requested, we should update the `useDataflow` field to an enum in that case. Because in general fif i request the cfg without call information i may have certain expectations (e.g. on the existence of resolved call vertices) - so this should not happen just because we can!